toolkit/tools/imagecustomizerapi/packages.go

```diff
@@ -4,11 +4,11 @@
 package imagecustomizerapi
 
 type Packages struct {
-	UpdateExistingPackages bool     `yaml:"updateExistingPackages"`
-	InstallLists           []string `yaml:"installLists"`
-	Install                []string `yaml:"install"`
-	RemoveLists            []string `yaml:"removeLists"`
-	Remove                 []string `yaml:"remove"`
-	UpdateLists            []string `yaml:"updateLists"`
-	Update                 []string `yaml:"update"`
+	UpdateExistingPackages bool     `yaml:"updateExistingPackages" json:"updateExistingPackages,omitempty"`
+	InstallLists           []string `yaml:"installLists" json:"installLists,omitempty"`
```

Comment:
The `validatePackageLists` function merges the packages from `installLists`, `removeLists`, and `updateLists` into `install`, `remove`, and `update`. This is probably useful from a history perspective. You could potentially omit the `installLists`, `removeLists`, and `updateLists` fields from the JSON since they will never be populated.

Reply:
omitted